change/msal-9dc2773a-eef8-4657-aa20-b98b34f8449f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "feat: adding the metadata npm commands",
```

Comment:
Please remove package-lock changes and remove this file

Reply:
I have removed this change file too.